server/server.py

```diff
@@ -94,16 +96,23 @@ def clusterplot(plot_type, variant):
         arr[0] = 'X' if arr[0] == '23' else arr[0] 
         exists_in_chip = fetch.check_var_in_chip(variant)
         filename = config['cluster_plots_location'] + '/' + plot_type + '/' + '_'.join(arr) + '.png'
-        with open(filename, 'rb') as f:
-            blob = f.read()
+        if (config['use_gcp_buckets']):
+            blob = cloud_storage.read_file(config['cluster_plots_bucket'], filename)
+            if blob is None:
+                raise FileNotFoundError("Requested cluster plot not found!")
```

Comment:
should we abort here instead of raise like on the lines 109/112/114

Reply:
I wanted to use the same error message as on 112/114. This raises an exception which is then caught on line 110.